ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -137,6 +140,12 @@ class TFLiteInterpreter
   int getTensorDim (int tensor_idx, tensor_dim dim);
   int setTensorProp (const std::vector<int> &tensor_idx_list,
       GstTensorsInfo * tensorMeta);
+
+  TfLiteDelegate* delegate_ = nullptr; /**< The delegate for tflite interpreter */
```

Comment:
Do you really need this pointer? It seems to be same with `stateful_nnapi_delegate.get ()`.

Reply:
I declare this pointer for multiple delegate support: NNAPI / gpu / ... .
I'll update this PR reflecting Jaeyun's comment:

> Discussion (tf-lite gelegate) for future work
> 
> * define enum (delegate : nnapi / gpu / ..) in tf-lite sub-plugin and set proper TfLiteDelegate
> * change custom option str (UseNNAPI:true > Delegate:NNAPI)